src/Zynga/Framework/IO/Web/V1/ManagerTest.hh

```diff
@@ -0,0 +1,30 @@
+<?hh // strict
+
+namespace Zynga\Framework\IO\Web\V1;
+
+use Zynga\Framework\Testing\TestCase\V2\Base as TestCase;
+use Zynga\Framework\Type\V1\UrlBox;
+
+class ManagerTest extends TestCase {
+  
+  public function testMakeCurlRequestWithMock(): void {
```

Comment:
For test names, can we change these to make the test specify the condition we are looking for? In this case, the test name doesn't immediately let us know what condition the code should operate under.

Also, can we spread out these asserts to only assert on 1 concept/condition? Have a giant list of asserts in test methods makes it hard to say what specific case is the unit test looking for.

Reply:
I will update the function name to make more sense. 
In terms of multiple asserts if, i will change this function name to be: 
testValidMockedCurlResponse - it should test the full response model which can mean having multiple assert statements. 

> Have a giant list of asserts in test methods makes it hard to say what specific case is the unit test looking for.
Having a clear function name should help in this case. Let me know what you think when i update the code

Reply:
With regards to 'testValidMockedCurlResponse', to me that is still not clear what the condition you are looking for is. Can we rephrase it so that we can state what the state of the output is for what you are looking for? For example:

```
testMockedCurlResponseReturnsExpectedValue(){}